packages/walletconnect-v2/src/index.ts

```diff
@@ -23,6 +23,19 @@ export type WalletConnectOptions = Omit<Parameters<typeof WalletConnectProvider.
   rpc?: { [chainId: number]: string | string[] }
 }
 
+/**
+ * Necessary type to interface with @walletconnect/ethereum-provider@2.9.2 which is currently unexported
```

Comment:
You should make an upstream issue/PR to have that exported.

Reply:
Talked to them about it when I first started working on this PR and they said they would take care of exporting it. May be worth checking in again.